azurelinuxagent/common/osutil/default.py

```diff
@@ -1439,6 +1439,31 @@ def _get_proc_stat():
 
         return results
 
+    @staticmethod
+    def get_system_uptime():
+        """
+        Get the uptime of the system (including time spent in suspend) in seconds.
+        /proc/uptime contains two numbers (values in seconds): the uptime of the system (including time spent in
+        suspend) and the amount of time spent in the idle process:
+            # cat /proc/uptime
+            365380.48 722644.81
+
+        :return: System uptime in seconds
+        :rtype: float
+        """
+        results = 0.0
+        try:
+            uptime_contents = fileutil.read_file('/proc/uptime').split()
+            results = float(uptime_contents[0])
+        except (OSError, IOError) as ex:
```

Comment:
logging should be done by the caller. since this is a utility function, as code evolves it may end up being called on operations with high frequency